ui/ducks/send/send.js

```diff
@@ -571,7 +571,11 @@ const slice = createSlice({
       state.recipient.nickname = action.payload.nickname;
       state.draftTransaction.id = action.payload.id;
       state.draftTransaction.txParams.from = action.payload.from;
-      slice.caseReducers.updateDraftTransaction(state);
+      if(state.asset.type === ASSET_TYPES.NATIVE){
```

Comment:
@brad-decker can we take a look at this PR together sometime today? I've spent too much time trying to figure out whats going wrong. The navigate back to edit flow is behaving inconsistently for me